src/modules/pivotTable/PivotTableEngine.js

```diff
@@ -103,6 +119,23 @@ const buildDimensionLookup = (visualization, metadata, headers) => {
         return out
     }, {})
 
+    const primaryDimensions = {
+        data: getPrimaryDimension(metadata, 'dx'),
+        period: getPrimaryDimension(metadata, 'pe'),
+        orgUnit: getPrimaryDimension(metadata, 'ou'),
```

Comment:
Is it worth using the constants that are defined here: modules/predefinedDimensions.js?

Reply:
Sure, will do

Reply:
Done